content/ri/using-redisinsight/troubleshooting.md

```diff
@@ -27,10 +27,10 @@ You can install RedisInsight on operating systems that are not officially suppor
 
 We are happy to receive your feedback at redisinsight@redislabs.com.
 
-
 ## Using behind a reverse proxy
 
-RedisInsight should work fine behind a reverse proxy like Nginx for most use-cases.
+RedisInsight should work fine behind a reverse proxy like NGINX for most use-cases.
```

Comment:
I just don't know - NGINX or Envoy?

Reply:
This is just giving an example of a proxy that you can setup RedisInsight behind. I think NGINX is fine.

Reply:
I'm more concerned this sentence: "RedisInsight should work fine behind a reverse proxy like NGINX for most use-cases." It's too vague "should work fine...for most use cases". Can we make this clearer?

Reply:
@slemeur Can you provide more detail about using RedisInsight behind a reverse proxy?